delfin/db/sqlalchemy/models.py

```diff
@@ -129,6 +129,29 @@ class Disk(BASE, DelfinBase):
     capacity = Column(BigInteger)
 
 
+class Port(BASE, DelfinBase):
+    """Represents a port object."""
+    __tablename__ = 'ports'
+    id = Column(String(36), primary_key=True)
+    native_port_id = Column(String(255))
```

Comment:
Similar comment for column size as was there for PR#407. We should restrict the size of the column

Reply:
Raised an issue for this (https://github.com/sodafoundation/delfin/issues/434) and will be handled separately.